Aptoide/aptoide-client-v8
# [Task/ASV-1545] Deprecate direct notifications

**What does this PR do?**

   Removes all the social (direct) campaigns related requests and configurations because the API was deprecated.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] NotificationSyncFactory.java

**How should this be manually tested?**

  Open Aptoide and check if direct notifications are not called nor App malfunctions.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1545](https://aptoide.atlassian.net/browse/ASV-1545)

**Code Review Checklist**
